src/pydvl/utils/parallel/map_reduce.py

```diff
@@ -54,33 +58,42 @@ def get_value(v: Union[ObjectRef, Iterable[ObjectRef], Any], *, timeout: int = 3
 
 
 class MapReduceJob(Generic[T, R]):
-    """Takes an embarrassingly parallel fun and runs it in num_jobs parallel
+    """Takes an embarrassingly parallel fun and runs it in n_jobs parallel
     jobs, splitting the data into the same number of chunks, one for each job.
 
     It repeats the process num_runs times, allocating jobs across runs. E.g.
-    if num_jobs = 90 and num_runs=10, each whole execution of fun uses 9 jobs,
-    with the data split evenly among them. If num_jobs=2 and num_runs=10, two
+    if n_jobs = 90 and num_runs=10, each whole execution of fun uses 9 jobs,
```

Comment:
Minor nits: `num_runs` should also be renamed to `n_runs`

Reply:
```suggestion
    if n_jobs = 90 and n_runs=10, each whole execution of fun uses 9 jobs,
```